cloud-service-providers/google-cloud/gke/variables.tf

```diff
@@ -546,7 +552,7 @@ variable "repository" {
 variable "model_name" {
   type        = string
   description = "Name of the NIM model"
-  default     = "meta/llama3-8b-instruct​"
+  #default     = "llama3-8b-instruct​"
```

Comment:
Why don't we need a default here any longer? When run from the UI we probably override this, but when run via `terraform apply` do we still want a default?

Reply:
We could set a default, `terraform apply` will ask for a value for variables that have no default. What i can do is set default values for a "path" of variables that and set `default = ""` for variables that won't be used in that path.